README.rst

```diff
@@ -137,5 +135,5 @@ version 3 of the License, or (at your option) any later version. See the files
 licensing conditions.
 
 
-.. |build status| image:: https://img.shields.io/github/workflow/status/dftbplus/skprogs/CI
-    :target: https://github.com/dftbplus/skprogs/actions/
+.. |build status| image:: https://img.shields.io/github/actions/workflow/status/dftbplus/skprogs/build.yml
+    :target: https://github.com/vanderhe/fortnet/actions/
```

Comment:
Why is fortnet the target?

Reply:
Surreptitious advertising..sorry, copy&paste error.